go.mod

```diff
@@ -1,8 +1,16 @@
 module github.com/oiime/logrusbun
 
-go 1.16
+go 1.20
```

Comment:
the `go` directive indicates the _minimum_ version required for the package, there is no reason to exclude older versions of go